secure_ec2/cli.py

```diff
@@ -102,7 +111,7 @@ def main(
                 "type": "rawlist",
                 "name": "keypair",
                 "message": "KeyPair",
-                "choices": keypairs + ["None"],
+                "choices": keypairs + ["None (Session Manager)"],
```

Comment:
OK I see what you did here. I guess I have to noodle on this but maybe it's ok. I feel it might be better just have a `--session_manager` argument because it allows for 1) a user to still optionally provide no keypair without SSM while 2) making it more explicit.

I guess this makes sense, or did I totally miss the point here?

Reply:
@avishayil I see it seems I was able to make it work with:

```sh
$ secure_ec2 --os_type Linux --num_instances 1 --instance_type t2.micro --keypair 'None (Session Manager)'
```

Not sure that is the final intent, but that was some PEBKAC, read the manual stuff on my part, my bad!

Reply:
Good catch.
I tried to make the wizard more clear, but clearly it's not intended.
I'll try to think about `session_manager` argument like you suggested. However, we might need to fail / warn if there is no KeyPair and no `session_manager` param (The user won't be able to connect).

Reply:
@xee5ch I removed the session manager text that caused the confusion

Reply:
@xee5ch let's open another issue for the `session_manager` argument

Reply:
I needed to set up a new burner VPC in my test org to play with this and I went down that rabbit hole Sunday. Some work came up, so i will look at this tomorrow. Rest assured, I have not forgotten!